BookReaderDemo/demo-plugin-menu-toggle.html

```diff
@@ -18,7 +18,8 @@
     <link rel="stylesheet" href="../BookReader/BookReader.css"/>
     <script src="../BookReader/BookReader.js"></script>
     <script type="text/javascript" src="../BookReader/plugins/plugin.url.js"></script>
-    <script type="text/javascript" src="../BookReader/plugins/plugin.fullscreen_menu_toggle.js"></script>
+    <!-- <script type="text/javascript" src="../BookReader/plugins/plugin.menu_toggle.js"></script> -->
```

Comment:
QUESTION: Did you want this commented out line to remain in the codebase?

Reply:
no, sorry, just artifact with testing...